tools/rpi-sign-bootcode

```diff
@@ -133,18 +158,18 @@ class ImageFile:
                 raise Exception("Bad key length %d expected %d" % (len(hmac_key), expected_keylen))
 
         if digest_alg == 'hmac-sha256':
-            digest = HMAC.new(base64.b16decode(hmac_key, True), self._bytes, digestmod=SHA256)
+            h = HMAC.new(base64.b16decode(hmac_key, True), self._bytes, digestmod=SHA256)
         elif digest_alg == 'hmac-sha1':
-            digest = HMAC.new(base64.b16decode(hmac_key, True), self._bytes, digestmod=SHA1)
-        elif digest_alg == 'sha256':
-            digest = SHA256.new(self._bytes)
-        elif digest_alg == 'sha1':
-            digest = SHA1.new(self._bytes)
+            h = HMAC.new(base64.b16decode(hmac_key, True), self._bytes, digestmod=SHA1)
+        elif digest == 'sha256':
```

Comment:
Is the change from `digest_alg` to `digest` here correct? (from a quick glance, the `digest` variable doesn't appear to be in scope in this function?)